docs/contributing.rst

```diff
@@ -365,10 +365,13 @@ Now run tests inside the Docker image:
 Notes on OS dependencies
 ========================
 
-OS level dependencies are managed by scripts in ``bootstrap``.  Some notes
-are provided here mainly for the :ref:`developers <hacking>` reference.
+OS-level dependencies can be installed like so:
 
-In general:
+.. code-block:: shell
+
+    letsencrypt-auto/letsencrypt-auto --os-packages-only
```

Comment:
Actually, I think we should just make it simpler and revert all changes to this file for now.